SU-HSDO/suhumsci
# HSD8-1473: Add News RSS bundle

# READY FOR REVIEW

## Summary
- Added `hs_entities` module with new `hs_entity` Entity Type
- Added News RSS bundle to `hs_entity` Entity Type
- Removed News RSS ECK bundle and Importers collections ECK Entity type
- Added update hooks to facilitate all of the above and migrate relevant data from the old to the new

## Need Review By (Date)
_['10/30', 'asap', etc.]_

## Urgency
_['low', 'medium', 'high', etc.]_

## Steps to Test
1. Checkout the latest release.
    - `git checkout 9.7.1-release`
1. Install the default site locally
    - `drush si su_humsci_profile -y`
1. Create a few News RSS ECK items
    - Create here: `/admin/structure/migrate/news-rss`
1. Checkout this branch
    - `git checkout HSD8-1473--add-news-rss-bundle`
1. Run database updates
    - `drush updatedb -y`
1. Verify
    - All the News RSS items you created still exist and have been migrated to the new Entity type (`/admin/structure/migrate/news-rss`).
    - The News RSS ECK Bundle has been removed.
    - The Importers Collections ECK Entity type has been removed (`/admin/structure/eck`).


## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
